### PR TITLE
Issue #107: Marked internal methods and variables as protected

### PR DESCRIPTION
**Pull request by deepthinidwannaya**
Marked internal variables and methods as protected

### CODE REVIEW

**Comment by SunyataZero** [comment]
@deepthinidwannaya Thank you for working on this, i've made a few suggestions using the github review/comment feature, can you look at this? Thanks again! /Tord

File: mc/gui/breathing_dlg.py [hunk 1]
```diff
@@ -17,10 +17,10 @@ class BreathingDlg(QtWidgets.QFrame):
...
     def __init__(self):
         super().__init__()
-        self.hover_active_bool = False
-        self.keyboard_active_bool = True
-        self.state = mc.mc_global.BreathingState.inactive
-        self.cursor_qtimer = None
+        self._hover_active_bool = False
+        self._keyboard_active_bool = True
+        self._cursor_qtimer = None
+        # TODO Remove unused fields?
```
[the remark this answers is not in the document]
Comment (deepthinidwannaya) [reply mc/gui/breathing_dlg.py]: Couldn't find ib_qtimer and ob_qtimer used

[the remark this answers is not in the document]
Comment (SunyataZero) [reply mc/gui/breathing_dlg.py]: Okay, you are right about ib_qtimer and ob_qtimer, i see now that i switched to using a qtimeline instead, please feel free to remove them if you want, then we can merge!

**Comment by SunyataZero** [comment]
@deepthinidwannaya Thank you, merged!

**Pull request merged**